src/wasm.rs

```diff
@@ -134,3 +138,139 @@ impl Server {
+        let want = JsValue::from_serde(csv_doc).unwrap();
+        let got = get_json_documentation("csv");
+        assert_eq!(want, got);
+    }
+
+    ///Retrieve full vector of PackageDoc documentation for entire stdlib
+    #[wasm_bindgen_test]
+    pub fn all_docs_test() {
+        let docs: Vec<PackageDoc> =
+            serde_json::from_slice(&docs_json().unwrap()).unwrap();
+        let mut got: PackageDoc = PackageDoc {
+            path: String::new(),
+            name: String::new(),
+            headline: String::new(),
+            description: None,
+            members: std::collections::HashMap::new(),
+            link: String::new(),
+        };
+        for d in docs {
+            if d.path == "array" {
+                got = d;
+                break;
+            }
+        }
```

Comment:
Constructing a blank doc and populating it seems unnecessarily complicated, and I don't think it's what we want to test anyway. We want to test that the doc at index 0 is what we expect it to be (which I guess is the `array` package?) and that the doc at index `docs.len()-1` is what we expect it to be. Looping over the docs until we find the array package, and then confirming that it's the array package, doesn't do us much good in terms of testing.

Also, I think it's enough to test against the path of the `got` value. It may be too much effort to assert on the deeper contents of the docs, given that we have tests elsewhere that do this.